src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -701,8 +701,8 @@ function copyOptions() {
 
     // Number of recipient setup
     formatNumberBox($currRecipient.val(), NEW_QUESTION);
-    var $prevRadioButtons = $('table[class~="questionTable"]').eq(-2).find('input[name="numofrecipientstype"]');
-    var $currRadioButtons = $('table[class~="questionTable"]').last().find('input[name="numofrecipientstype"]');
+    var $prevRadioButtons = $('div[class~="questionTable"]').eq(-2).find('input[name="numofrecipientstype"]');
```

Comment:
I've investigated the use of `.eq(` and it really seems like a good a consistent solution that's in use throughout the script. Currently, each question has a `questionTable` `div` that has an ID based on the question number of the existing question, e.g. `questionTable-1` for question 1, `questionTable-2` for question 2 and `questionTable--1` for the new question. Each of these `div`s is also tagged with the `questionTable` class.

Consequently, selecting the last existing question by ID requires knowledge of the number of existing questions. It seems much neater to select by class, then use `.eq(-2)` to pick the second last element, rather than writing additional code to figure out how many questions we have just to generate the correct ID and select the element.

If this really is unacceptable, I believe it belongs in another issue altogether, since the `.eq(-2)` and `.last()` pattern occurs in multiple places throughout the script. Should I open another issue to refactor `instructorFeedbackEdit.js`?